doc-content/enterprise-only/fuse/installing-on-fuse-eap-proc.adoc

```diff
@@ -1,20 +1,13 @@
 [id='installing-on-fuse-eap-proc']
 = Installing {FUSE} on {EAP_LONG}
-Install {FUSE_LONG} {FUSE_VERSION} on {EAP} 7.2.7 to use with {PRODUCT} to deploy integrated services where required.
-
-[NOTE]
-====
-{FUSE_LONG} 7.7 is supported on {EAP} 7.2, whereas {PRODUCT} 7.8.x and newer is supported on Red Hat {EAP} 7.3. To ensure a stable deployment environment, install {FUSE} and {PRODUCT} on different instances of {EAP} according to the supported {EAP} versions.
-
-{PRODUCT} 7.8.x and newer components on {EAP} 7.2 are not officially supported. However, Red Hat provides guidance on a commercially reasonable basis until {FUSE} is available on {EAP} 7.3. `MarshallingFormat.JSON` is not supported on {FUSE} integration with {EAP} 7.2 using `kie-camel`. The default is JAXB.
-====
+Install {FUSE_LONG} {FUSE_VERSION} on {EAP} {FUSE_EAP_VERSION} to use with {PRODUCT} to deploy integrated services where required.
```

Comment:
I'm struggling with this sentence. Seems like something's missing or out of sorts. Maybe this?

>Install {FUSE_LONG} {FUSE_VERSION} on {EAP} {FUSE_EAP_VERSION} to use {FUSE} with {PRODUCT} and deploy integrated services where required.

OR

> Install {FUSE_LONG} {FUSE_VERSION} on {EAP} {FUSE_EAP_VERSION} to deploy integrated services in {PRODUCT} where required.

Or something.

Reply:
Simplified it even further. Might still be a bit awkward but I think we can do a bigger style improvement of the entire document at a later stage, when we have time to have SME look at it. (This phrase was in the document, it is not an addition).